gcp/make_vm

```diff
@@ -119,6 +121,10 @@ gcloud compute instances create "${vm_name}" \
 ${preemptible_arg} \
 ${startup_script_arg} || die "Something wrong with VM creation. Check the gcloud response."
 
+labels=user=$USER,cost_center=${cost_center}
```

Comment:
This line doesn't look correct to me without double quotes. I don't think it matters that much actually but looks "wrong".